server/plugin/command.go

```diff
@@ -552,12 +589,23 @@ func getAutocompleteData(config *Configuration) *model.AutocompleteData {
 	subscriptionsAdd.AddTextArgument("Owner/repo to subscribe to", "[owner/repo]", "")
 	subscriptionsAdd.AddTextArgument("Comma-delimited list of one or more of: issues, pulls, pushes, creates, deletes, issue_creations, issue_comments, pull_reviews, label:\"<labelname>\". Defaults to pulls,issues,creates,deletes", "[features] (optional)", `/[^,-\s]+(,[^,-\s]+)*/`)
 	if config.GitHubOrg != "" {
-		flags := []model.AutocompleteListItem{{
-			HelpText: "Events triggered by organization members will not be delivered (the organization config should be set, otherwise this flag has not effect)",
-			Hint:     "(optional)",
-			Item:     "--exclude-org-member",
-		}}
-		subscriptionsAdd.AddStaticListArgument("Currently supports --exclude-org-member", false, flags)
+		exclude := []model.AutocompleteListItem{
+			{
+				HelpText: "notification for these repo will be turned off",
+				Hint:     "(optional)",
+				Item:     "--exclude",
+			},
+		}
+		subscriptionsAdd.AddStaticListArgument("Currently supports --exclude", true, exclude)
+		subscriptionsAdd.AddTextArgument("Owner/repo to subscribe to", "[owner/repo]", "")
+		flags := []model.AutocompleteListItem{
+			{
+				HelpText: "Events triggered by organization members will not be delivered (the organization config should be set, otherwise this flag has not effect)",
```

Comment:
```suggestion
				HelpText: "Events triggered by organization members will not be delivered (the organization config should be set, otherwise this flag has no effect)",
```

Reply:
Done